zeebe/gateway-protocol/src/main/proto/v2/audit-logs.yaml

```diff
@@ -0,0 +1,505 @@
+## Endpoint definitions
+paths:
+  /audit-logs/search:
+    post:
+      x-eventually-consistent: true
+      tags:
+        - Audit Log
+      operationId: searchAuditLogs
+      summary: Search audit logs
+      description: Search for audit logs based on given criteria.
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AuditLogSearchQueryRequest'
+      responses:
+        "200":
+          description: The audit logs search result.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AuditLogSearchQueryResult'
+        "400":
+          $ref: 'common-responses.yaml#/components/responses/InvalidData'
+        "401":
+          $ref: 'common-responses.yaml#/components/responses/Unauthorized'
+        "403":
+          $ref: 'common-responses.yaml#/components/responses/Forbidden'
+        "500":
+          description: An internal error occurred while processing the request.
+
+  /audit-logs/{auditLogKey}:
+    get:
+      x-eventually-consistent: true
+      tags:
+        - Audit Log
+      operationId: getAuditLog
+      summary: Get audit log
+      description: Get an audit log by auditLogKey.
+      parameters:
+        - name: auditLogKey
+          in: path
+          required: true
+          description: The audit log key.
+          schema:
+            $ref: 'keys.yaml#/components/schemas/AuditLogKey'
+      responses:
+        "200":
+          description: The audit log is successfully returned.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AuditLogResult'
```

Comment:
❓ @danielkelemen we have a requirement that, if audit log tracking is disabled, we return a message that describes how to configure it, instead of an error response.

Should we model it now, or do we schedule it  when we add the configuration properties?